### ISSUE TITLE
[FR] Material specific start and end g-code scripts

### ISSUE THREAD

**Issue by kulfuerst** [smartavionics/Cura#35]
I've started printing polypropylene and Nylon and both of these materials are different enough from the typical PLA or PETG that custom start and end g-code scripts would be extremely useful. I understand that this may not be possible to implement, but if it somehow was, this would be of great help.

I love your build which is pretty much the best thing to happen to my 3D printing endeavors. wishing you the best :)

**Comment by x0rtrunks** [comment]
Yes this would be a nice feature. I miss having the filament gcode option from Slic3r. I miss the filament library in general actually. Is it just me, or is the Cura filament manager basically useless? 

Anyway, I digress. I do wish there were more gcode scripting options available in general.
